### PR TITLE
this source has an error that makes to stay charging at page 10

### PR DESCRIPTION
**Pull request by poliglesias**
fix ratelimit issue

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by alessandrojean** [comment]
Please follow the PR checklist strictly...

> Updated `extVersionCode` value in `build.gradle` for individual extensions

**Comment by arkon** [comment]
Converting to a draft PR for now, please mark it as ready for review once you've updated it.

File: src/es/lectormanga/src/eu/kanade/tachiyomi/extension/es/lectormanga/LectorManga.kt [hunk 1]
```diff
@@ -565,7 +565,7 @@ class LectorManga : ConfigurableSource, ParsedHttpSource() {
         private const val IMAGE_CDN_RATELIMIT_PREF_SUMMARY = "Este valor afecta la cantidad de solicitudes de red para descargar imágenes. Reducir este valor puede disminuir errores al cargar imagenes, pero la velocidad de descarga será más lenta. Se requiere reiniciar Tachiyomi. \nValor actual: %s"
         private const val IMAGE_CDN_RATELIMIT_PREF_DEFAULT_VALUE = "10"
 
-        private val ENTRIES_ARRAY = (1..10).map { i -> i.toString() }.toTypedArray()
+        private val ENTRIES_ARRAY = listOf(1, 2, 3, 5, 6, 7, 8, 9, 10, 15, 20, 30, 40, 50, 100).map { i -> i.toString() }.toTypedArray()
```
[the remark this answers is not in the document]
Comment (poliglesias) [reply src/es/lectormanga/src/eu/kanade/tachiyomi/extension/es/lectormanga/LectorManga.kt:568]: in other extensions it works fine with out "

**Comment by alessandrojean** [comment]
This PR is not yet ready to merge, you still need to bump the `extVersionCode` as told earlier...

**Comment by poliglesias** [comment]
`apply plugin: 'com.android.application'
apply plugin: 'kotlin-android'

ext {
    extName = 'LectorManga'
    pkgNameSuffix = 'es.lectormanga'
    extClass = '.LectorManga'
    extVersionCode = 23
    isNsfw = true
}

dependencies {
    implementation project(':lib-ratelimit')
}

apply from: "$rootDir/common.gradle"`

**Comment by CarlosEsco** [comment]
Is this good? @alessandrojean

**Comment by alessandrojean** [comment]
Looks like so, yeah.

**Pull request merged**